plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/web/config/ConfigController.java

```diff
@@ -221,38 +235,80 @@ public void setConfigValidator(ConfigValidator configValidator) {
     public ResponseEntity<RestResponse> getCaseDefinitions(@PathVariable String configCode, @PathVariable String deploymentUnit) {
 
         JSONObject cases = null;
-        try{
+        try {
 
             KieServerConfigDto serverConfigDto = kieConfigService.getConfig(configCode);
             KieBpmConfig bpmConfig = kieConfigService.buildConfig(serverConfigDto);
             //TODO This call makes the fetched config the active one.  This should be scoped to the widget at some point
             caseManager.setConfig(bpmConfig);
 
-            cases = ((CaseManager)caseManager).getCasesDefinitions(deploymentUnit);
-        }catch(Exception e) {
-            logger.error("Failed to fetch container ids ",e );
+            cases = ((CaseManager) caseManager).getCasesDefinitions(deploymentUnit);
+        } catch (Exception e) {
+            logger.error("Failed to fetch container ids ", e);
         }
 
-        if(cases == null) {
+        if (cases == null) {
             cases = new JSONObject();
         }
         return new ResponseEntity<>(new RestResponse(cases.toMap()), HttpStatus.OK);
     }
 
-
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(value = "/kiebpm/serverConfigs/{configCode}/processesList/{processId:.+}/overrides", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<RestResponse> getProcessOverrides(@PathVariable String configCode, @PathVariable String processId) {
 
         List<KieFormOverride> overrides = new ArrayList<>();
-        try{
+        try {
             overrides = overrideManager.getFormOverrides(configCode, processId);
-        }catch(Exception e) {
-            logger.error("Failed to fetch container ids ",e );
+        } catch (Exception e) {
+            logger.error("Failed to fetch container ids ", e);
         }
 
         return new ResponseEntity<>(new RestResponse(overrides), HttpStatus.OK);
     }
 
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(value = "/kiebpm/datatableWidget/{configId}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<RestResponse> getDataTableWIdgetConfig(@PathVariable int configId) {
+
+        DatatableWidgetConfigDto dto = this.getKieBpmService().getDataTableWIdgetConfig(configId);
```

Comment:
Add controller level logging to all of these methods at debug. Both input and output

Reply:
added.